services/auth_service.py

```
@@ -0,0 +1,164 @@
+    if query_text:
+        # 用户名模糊搜索
+        or_conditions.append({
+            "UserName": {"$regex": re.escape(query_text), "$options": "i"}
+        })
+
+        # 邮箱模糊搜索
+        or_conditions.append({
+            "email": {"$regex": re.escape(query_text), "$options": "i"}
+        })
+
+        # _id 搜索（支持完整或前缀）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)